src/components/organisms/AdminVenueView/components/Spaces/Spaces.tsx

```diff
@@ -0,0 +1,246 @@
+import React, { useCallback, useState } from "react";
+import { useHistory } from "react-router";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faCaretRight,
+  faCaretDown,
+  faHome,
+} from "@fortawesome/free-solid-svg-icons";
+
+import { VenueTemplate, Venue_v2 } from "types/venues";
+import { RoomData_v2, RoomTemplate, VenueRoomTemplate } from "types/rooms";
+import { Dimensions, Position } from "types/utility";
+
+import { BackgroundSelect } from "pages/Admin/BackgroundSelect";
+import { VenueRoomItem } from "components/molecules/VenueRoomItem";
+import { EditRoomForm } from "components/molecules/EditRoomForm";
+import { MapPreview } from "components/organisms/AdminVenueView/components/MapPreview";
+
+import RoomIconConversation from "assets/icons/icon-room-conversation.svg";
+import RoomIconAuditorium from "assets/icons/icon-room-auditorium.svg";
+import RoomIconMusicBar from "assets/icons/icon-room-musicbar.svg";
+import RoomIconBurnBarrel from "assets/icons/icon-room-burnbarrel.svg";
+import RoomIconArtPiece from "assets/icons/icon-room-artpiece.svg";
+import RoomIconExperience from "assets/icons/icon-room-experience.svg";
+import RoomIconExternalLink from "assets/icons/icon-room-externallink.svg";
+import RoomIconMap from "assets/icons/icon-room-map.svg";
+
+import "./Spaces.scss";
+
+interface VenueRooms {
+  text: string;
+  template?: VenueRoomTemplate;
+  icon: string;
+}
+
+const venueRooms: VenueRooms[] = [
+  {
+    text: "Conversation Space",
+    icon: RoomIconConversation,
+    template: VenueTemplate.conversationspace,
+  },
+  {
+    text: "Auditorium",
+    icon: RoomIconAuditorium,
+    template: VenueTemplate.audience,
+  },
+  {
+    text: "Music Bar",
+    icon: RoomIconMusicBar,
+    template: VenueTemplate.jazzbar,
+  },
+  {
+    text: "Burn Barrel",
+    icon: RoomIconBurnBarrel,
+    template: VenueTemplate.firebarrel,
+  },
+  {
+    text: "Art Piece",
+    icon: RoomIconArtPiece,
+    template: VenueTemplate.artpiece,
+  },
+  {
+    text: "Experience",
+    icon: RoomIconExperience,
+    template: VenueTemplate.zoomroom,
+  },
+  {
+    text: "External link",
+    icon: RoomIconExternalLink,
+    template: RoomTemplate.external,
+  },
+  {
+    text: "Map",
+    icon: RoomIconMap,
+    template: VenueTemplate.partymap,
+  },
+];
+
+export interface SpacesProps {
+  venue: Venue_v2;
+  onClickNext: () => void;
+}
+
+export const Spaces: React.FC<SpacesProps> = ({ venue, onClickNext }) => {
+  const [selectedRoom, setSelectedRoom] = useState<RoomData_v2>();
+  const [showRooms, setShowRooms] = useState<boolean>(false);
```

Comment:
useShowHide?

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1583/commits/f8fa0c67c34ac626919737b4bbd112b4bd715553